src/Microsoft.AspNet.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -160,8 +168,13 @@ public virtual ConflictResult Conflict()
         /// <param name="value">The content value to negotiate and format in the entity body.</param>
         /// <returns>A <see cref="NegotiatedContentResult{T}"/> with the specified values.</returns>
         [NonAction]
-        public virtual NegotiatedContentResult<T> Content<T>(HttpStatusCode statusCode, [NotNull] T value)
+        public virtual NegotiatedContentResult<T> Content<T>(HttpStatusCode statusCode, T value)
         {
+            if (value == null)
```

Comment:
@pranavkm what does "works at runtime" mean? Does it throw if `value == default(T)` for example?

Not surprising there's no compilation warning. Compilers tend to be silent unless they're _sure_ you're doing something wrong.